.github/workflows/hassfest.yaml

```
@@ -0,0 +1,16 @@
+name: Validate with hassfest
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
```

Try to be consistent with double quotes: you used single quotes here, but later you used double one. I do understand the reason behind it: you copied it from the [official documentation](https://github.com/home-assistant/actions?tab=readme-ov-file#hassfest). However, if you do that, make sure that the code is correct and its quality is acceptable:

```suggestion
    - cron: "0 0 * * *"
```